deepspeed/runtime/zero/stage_1_and_2.py

```diff
@@ -519,6 +519,9 @@ def __init__(self,
         self._enable_universal_checkpoint()
         self._param_slice_mappings = self._create_param_mapping()
 
+    def destroy(self):
+        self._uncreate_param_mapping()
```

Comment:
Don't we need to delete the buffers related to contiguous_gradients and gradient_partitions?

Reply:
Deleting those buffers didn't make a difference when running experiments but I can add a full delete of all attributes for completeness just as I did for ZeRO 3.

Reply:
Sounds good. Is it possible to make the attribute deletion function a utility that lives in deepspeed.utils?